crates/astria-sequencer/src/assets/storage/keys.rs

```diff
@@ -0,0 +1,118 @@
+use std::borrow::Cow;
+
+use astria_core::primitive::v1::asset::IbcPrefixed;
+use astria_eyre::eyre::{
+    bail,
+    ContextCompat,
+    Result,
+    WrapErr,
```

Comment:
```suggestion
    WrapErr as _,
```

Reply:
Done in [1c8f7d1](https://github.com/astriaorg/astria/pull/1616/commits/1c8f7d1ed51e245cb547187ae21510214e3cd490).